---
title: "Epigenetic and chromosomal features drive transposon insertion in Drosophila melanogaster"
collection: publications
category: manuscripts
permalink: /publication/2023-NAR-TE-Insertion-Preference
excerpt: 'Transposons are the major source of insertional structural variation, but their integration site is not randomly selected. This paper studied the preference of LTR, LINE, and DNA transposon integration site selection, reflecting their underlying replication mechanisms.'
date: 2023-05-01
venue: 'Nucleic Acid Research'
slidesurl: '/files/Paper-Sildes-2023-NAR.gif'
paperurl: 'https://academic.oup.com/nar/article/51/5/2066/7033800?login=false'
citation: 'Cao & Yu et al. "Epigenetic and chromosomal features drive transposon insertion in Drosophila melanogaster." Nucleic Acids Research 51.5 (2023): 2066-2086.'
---
Transposons are mobile genetic elements prevalent in the genomes of most species. The distribution of transposons within a genome reflects the actions of two opposing processes: initial insertion site selection, and selective pressure from the host. By analyzing whole-genome sequencing data from transposon-activated Drosophila melanogaster, we identified 43 316 de novo and 237 germline insertions from four long-terminal-repeat (LTR) transposons, one LINE transposon (I-element), and one DNA transposon (P-element). We found that all transposon types favored insertion into promoters de novo, but otherwise displayed distinct insertion patterns. De novo and germline P-element insertions preferred replication origins, often landing in a narrow region around transcription start sites and in regions of high chromatin accessibility. De novo LTR transposon insertions preferred regions with high H3K36me3, promoters and exons of active genes; within genes, LTR insertion frequency correlated with gene expression. De novo I-element insertion density increased with distance from the centromere. Germline I-element and LTR transposon insertions were depleted in promoters and exons, suggesting strong selective pressure to remove transposons from functional elements. Transposon movement is associated with genome evolution and disease; therefore, our results can improve our understanding of genome and disease biology.

<img src="/images/Paper-Cover-2023-NAR.png" alt="Graphic Abstract" width="370" height="400"/>